eosdac-material/src/i18n/ko/index.js

```diff
@@ -178,25 +178,25 @@ export default {
     transfer_to: '전송 대상',
     transfer_tokens: '토큰 전송',
     wallet: '지갑',
-    total_staked: '토탈 스테이 킹',
+    total_staked: '스테이킹한 토큰',
     send_tokens: '토큰 보내기',
-    total_eos_balance: '총 잔량 균형',
+    total_eos_balance: '총 EOS 잔액',
     liquid: '액체',
-    token_selection_and_quantity: '토큰 선택 및 수량',
+    token_selection_and_quantitAy: '토큰 선택 및 수량',
```

Comment:
Looks like an extra "A" got added in here. Could you go ahead and fix that? If you're using command line git tools, you can simple make a change and commit it to the existing branch you created and this PR will updated automatically. If not, no worries, I can make the change myself.

Thanks!